### DISCUSSION TITLE
i don't know the syntax: key in dict vs v := dict[key], which is faster?

### DISCUSSION THREAD

**Discussion by timi5691** [vlang/v#15143] [General]
mut dict1 := {1: 'one', 2: 'two', ....} // too many keys in the dict
mut dict2 := {1: 'mot', 2: 'hai', ....)

to check if number 3 in dict1 && dict2, i wrote:
if 3 in dict1 && 3 in dict2 {
        do_something()
}

i wonder if i can worte like this:
if _ := dict1[3] && _ := dict2[3] {
      do_something()
}

i don't know the syntax: key in dict vs v := dict[key], which is faster?

**Comment by David-Worboys** [comment]
I think maps is what you are after...

https://github.com/vlang/v/blob/master/doc/docs.md#maps
